roles/confluent.ssl/tasks/custom_certs.yml

```diff
@@ -1,16 +1,19 @@
 ---
 - name: Copy CA Cert to Host
   copy:
+    remote_src: "{{ssl_remote_certs | default(omit)}}"
```

Comment:
could also just default `ssl_custom_certs_remote_src: false` in a variables file